cmd/sansshell-server/default-policy.rego

```diff
@@ -68,3 +68,7 @@ denial_hints[msg] {
 	input.message.file.filename != "/etc/hosts"
 	msg := "we only allow /etc/hosts"
 }
+
+allow {
+	input.method = "/SysInfo.SysInfo/Uptime"
+}
```

Comment:
nit: add a newline

If you don't want to do this by hand, you can format by installing the opa cli and running `opa fmt -w`

Reply:
Oh, will keep that in mind